java/src/main/java/com/genexus/webpanels/GXOAuthAccessToken.java

```diff
@@ -164,25 +170,6 @@ protected void doExecute(HttpContext context) throws Exception
 				context.sendResponseStatus(404, e.getMessage());
 			}
    	}
-	
-   
-    
-    private JSONObject errorJson;
-    public void SetError(String code, String message)
```

Comment:
Ok to remove the "errorJson" field. 
But why remove this Method? 

Won't be better to change the method to a return method instead?
The code is not equivalent, as it had a Try Catch block. The new code will crash if the JSONObject cannot be built. 

```
private JSONObect getError(String code, String message) {

}

```

Reply:
Thought it was equivalent since the code is inside a try-catch block in the doExcecute method. I applied your proposed changes anyways